src/main/java/com/dumbdogdiner/stickyapi/common/util/textures/DefaultSkins.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util.textures;
+
+import lombok.Getter;
+
+public enum DefaultSkins {
```

Comment:
Move to resources please.

Reply:
What format should these be in in resources? SQLite DB??

Reply:
JSON.